src/core/agent/services/connectionService.ts

```diff
@@ -66,6 +62,7 @@ class ConnectionService extends AgentService {
   static readonly FAILED_TO_RESOLVE_OOBI =
     "Failed to resolve OOBI, operation not completing...";
   static readonly CANNOT_GET_OOBI = "No OOBI available from KERIA";
+  static readonly CONNECTION_NOTE_PREFIX_KEY = "note:";
```

Comment:
Can we have a string enum for this? Because then we can use eslint to make sure there are no duplicate prefixes for anything contact related on KERIA

Reply:
could be in a connectionService.types.ts file